examples_src/draw-features.js

```diff
@@ -14,7 +14,7 @@ var raster = new ol.layer.Tile({
   source: new ol.source.MapQuest({layer: 'sat'})
 });
 
-var source = new ol.source.Vector();
+var source = new ol.source.Vector({wrapX: false});
```

Comment:
If you rebase on master, I think the 3-way merge will do the right thing (make these changes in `examples` instead of `examples_src`).